js/assets/sourcePacks.json

```diff
@@ -70,6 +70,10 @@
         {
           "title": "ZucasKittens",
           "url": "https://www.instagram.com/zucaskittens/media/?hl=en"
+        },
+        {
+          "title": "youngestoldcatlady",
+          "url": "https://www.instagram.com/youngestoldcatlady/?hl=en"
```

Comment:
Should be https://www.instagram.com/youngestoldcatlady/media/?hl=e so that it outputs JSON instead of HTML.